core/src/main/java/com/linecorp/armeria/server/logging/structured/CommonServiceLogEntry.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2016 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.logging.structured;
+
+import java.util.concurrent.TimeUnit;
+
+import com.linecorp.armeria.common.RequestContext;
+import com.linecorp.armeria.common.logging.RequestLog;
+import com.linecorp.armeria.common.logging.ResponseLog;
+
+/**
+ * A representation and constructor of a service log which holds only very common fields that are protocol
+ * agnostic.
+ */
+public class CommonServiceLogEntry {
+    private final long timestampMs;
+    private final long responseTimeMs;
+
+
+    CommonServiceLogEntry(long timestampMs, long responseTimeMs) {
+        this.timestampMs = timestampMs;
+        this.responseTimeMs = responseTimeMs;
+    }
+
+    /**
+     * Constructs {@link CommonServiceLogEntry} from {@link RequestContext} and {@link ResponseLog}.
+     * Can be used as {@link ServiceLogEntryBuilder}.
+     */
+    public CommonServiceLogEntry(RequestContext reqCtx, ResponseLog resLog) {
```

Comment:
was intent to keep this constructor compatible as `ServiceLogEntryBuilder`(`ServiceLogEntry::new`)...
agree in advance that this isn't a good interface but couldn't imagine better way to do it with keeping all fields immutable.